funcbench/env.go

```diff
@@ -98,14 +99,23 @@ type GitHub struct {
 }
 
 func newGitHubEnv(ctx context.Context, e environment, owner, repo string, prNumber int) (Environment, error) {
-	if err := os.Chdir(os.Getenv("GITHUB_WORKSPACE")); err != nil {
-		return nil, errors.Wrap(err, "changing to GITHUB_WORKSPACE dir")
+	if e.home == "" {
+		e.home = os.Getenv("WORKSPACE")
```

Comment:
I think you can just move this out of the if block?

Reply:
Update it.
Basically it reads from `-w` flag first, then environment variable `WORKSAPCE`, in the end, set to current directory.